### ISSUE TITLE
[Feature Request] Add a check if Token is empty or not and show better error message

### ISSUE THREAD

**Issue by alexander-schranz** [peter-evans/create-pull-request#1321]
### Subject of the issue

I did run into https://github.com/peter-evans/create-pull-request/issues/871

```bash
fatal: could not read Username for 'https://github.com/': No such device or address
  Error: The process '/usr/bin/git' failed with exit code 128
```

And sadly github doesn't allow some debugging if all is given correctly to the action. It would be nice if the action could check if `token` is not empty and show a more friendly

### Steps to reproduce

Try to use the action without a token. The error is a little bit confusing.

**Comment by peter-evans** [comment]
Thank you for raising this. I've added a check for the `token` input and released as `v4.2.3` / `v4`.

**Comment by alexander-schranz** [comment]
@peter-evans that is really great thank you!
